WebhookNotificationActivity/index.ts

```diff
@@ -44,14 +53,26 @@ const notificationModel = new NotificationModel(
 // Whether we're in a production environment
 const isProduction = process.env.NODE_ENV === "production";
 
+// Webhook must be an https endpoint
+const abortableFetch = AbortableFetch(agent.getHttpsFetch(process.env));
+// 10 seconds timeout by default
+const fetchWithTimeout = setFetchTimeout(
```

Comment:
this behavior differs from the one we implemented in `app-backend` where we rely on `agentkeepalive`'s defaults, I would try to keep the behavior consistent across components

Reply:
what do you mean exactly ? the configure http agent does not terminate (abort) the connection, it only signal a timeout event and keeps the connection open. to close it we must use `setFetchTimeout`

Reply:
sorry I misunderstood the code